setup.py

```diff
@@ -150,5 +150,5 @@
         'magenta': ['models/image_stylization/evaluation_images/*.jpg'],
     },
     setup_requires=['pytest-runner', 'pytest-pylint'],
-    tests_require=['pytest', 'pylint'],
+    tests_require=['pytest', 'pylint < 2.0.0;python_version=="2.7"'],
```

Comment:
Does this maintain the requirement for py3? I'm not super familiar with how [environment markers](https://www.python.org/dev/peps/pep-0508/#environment-markers) work, but should it be something like:

```
'pylint < 2.0.0;python_version<"3"',
'pylint > 2.0.0;python_version>="3"',
```

Reply:
I'm not familiar with this syntax either. I think you're right, even though the missing part for py3 doesn't break tests for py3 environment. This is odd. Anyway, thank you for pointing out!